apps/omg_api/test/omg_api/integration/monitor_test.exs

```diff
@@ -77,70 +77,17 @@ defmodule OMG.API.MonitorTest do
   @tag :capture_log
   test "if a map spec child gets restarted after exit" do
     child = __MODULE__.Mock.prepare_child()
-    {:ok, monitor_pid} = Monitor.start_link([child])
+    {:ok, monitor_pid} = Monitor.start_link([Alarm, [child]])
     handle_killing_and_monitoring(monitor_pid)
   end
 
   @tag :capture_log
   test "if a tuple spec child gets restarted after exit" do
     child = {__MODULE__.Mock, []}
-    {:ok, monitor_pid} = Monitor.start_link([child])
+    {:ok, monitor_pid} = Monitor.start_link([Alarm, [child]])
     handle_killing_and_monitoring(monitor_pid)
   end
 
-  test "if a child gets a interval set after termination" do
```

Comment:
1/ can you give a quick 101, why these tests aren't needed anymore
2/ is there a test we can add to cover the behavior that was fixed in this fix? (not sure if here would be the location of such test)

Reply:
Yeah sure, so basically, because there's no timers, there's no race conditions. the monitor will only restart his children if he receives a gen_even event that the alarm was cleared. and previous tests test exactly that.